src/Smart.Design.Razor/TagHelpers/Titles/PageTitle/PageTitleTagHelper.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Smart.Design.Razor.TagHelpers.Constants;
+
+
+namespace Smart.Design.Razor.TagHelpers.Titles.PageTitle;
+
+[HtmlTargetElement(TagNames.PageTitleTagName)]
```

Comment:
I wanted to propose that to write simply `<smart-page-title title="Title">` without closing the tag
```suggestion
[HtmlTargetElement(TagNames.PageTitleTagName, TagStructure = TagStructure.WithoutEndTag)]
```
But although it works, the IDE doesn't pick up on it and will make it looks like an error by default (so let's forget about it, just writing it as a thought :b)